mlflow/langchain/databricks_dependencies.py

```diff
@@ -242,6 +242,11 @@ def _traverse_runnable(
         # Visit the returned graph
         for node in lc_model.get_graph().nodes.values():
             yield from _traverse_runnable(node.data, visited)
+
+        # Visit the variables of the function as well to extract dependencies
+        if hasattr(lc_model, "func"):
```

Comment:
Awesome we found a fix, QQ: how stable is this `lc_model.func` API?

Reply:
Not sure of how stable it is, but here we are focussing on types that are Runnable. When we pass a function to the runnable, it will be stored into the func attribute. The way we build agents in our generated code is, to wrap the agent in the function and then pass it to a runnable. So our generated code will always have a func variable.